### ISSUE TITLE
Forbid too deep attribute usages

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#738]
# Rule request

## Thesis

Code like:

```python
self.attr.inner.wrapper.field.value

self.attr.inner.wrapper.method.call()
```

Should be forbidden, related #737 
The default configuration for this complexity rule is `4`

## Reasoning

It indicates a bad design. Too deep attributes shows that your OOP is not so OOP.
This is a complexity rule.

**Comment by sobolevn** [comment]
Joined with #737
